be/src/exprs/vectorized/function_call_expr.h

```diff
@@ -31,8 +31,13 @@ class VectorizedFunctionCallExpr final : public Expr {
 private:
     const FunctionDescriptor* _fn_desc;
 
-    // is rand/random function.
-    bool _is_rand_function = false;
+    bool returns_random_value() const {
+        if (_fn.name.function_name == "rand" || _fn.name.function_name == "random" ||
```

Comment:
better to use function id to improve performance.
and give some comment for the function id.